pkg/hostagent/hostagent.go

```diff
@@ -527,7 +534,14 @@ sudo chown -R "${USER}" /run/host-services`
 			return errors.Join(unlockErrs...)
 		})
 	}
+
 	if !*a.instConfig.Plain {
+		staticPortForwards, err := a.separateStaticPortForwards()
+		if err != nil {
+			errs = append(errs, err)
+		}
+		a.addStaticPortForwardsFromList(ctx, staticPortForwards)
```

Comment:
`a.addStaticPortForwardsFromList()` is only called when `--plain=false`, so it does not work as intended when `--plain=true`.

Reply:
filed #4002